fastlane/Fastfile

```diff
@@ -92,6 +92,7 @@ platform :ios do
     check_no_git_tag_exists(version_number)
     check_pods
     carthage_archive
+    export_xcframework
```

Comment:
added this extra step so that we check that we can export xcframeworks correctly in the `deployment checks` job